app/assets/stylesheets/admin/composants/_lettrisme.scss

```
@@ -1,6 +1,6 @@
 .evaluation__restitution-globale {
   .sous-competence__container {
-    padding: 10px 0px;
+    padding: .5rem 0px;
     margin-top: 2rem;
```

On peut faire 
```suggestion
    margin: 2rem 0;
```
Plutôt que d'ajouter une règle sur `.lettrisme-sous-competences margin-bottom` comme tu le fais à la ligne 12

---

J'ai pris en compte cette proposition